bitcoind_mock/conf.py

```diff
@@ -0,0 +1,16 @@
+# Make a copy of this file and rename it as conf.py
```

Comment:
conf.py is a user file, shouldn't be here (I'll add a .gitignore including it after merging the PR). Can you rebase it out?